src/sidebar.js

```diff
@@ -369,225 +369,243 @@ module.exports = {
   ],
   selfhosting: [
     "ecosystem/projects",
-    "ecosystem/community",
-    "ecosystem/roadmap",
-    "ecosystem/changelog",
-    "ecosystem/versioning",
+    {
+      type: "category",
+      label: "Open source",
+      items: [
+        "open-source",
+        "ecosystem/community",
+        "ecosystem/contributing",
+        "open-source/commitment",
+        "ecosystem/software-architecture-philosophy",
+        {
+          type: "category",
+          label: "Guidelines",
+          items: ["open-source/guidelines/rest-api-guidelines"],
+        },
+      ],
+    },
+    "self-hosted/deployment",
     "ecosystem/configuring",
-    "ecosystem/deployment",
+    {
+      type: "category",
+      label: "Operations and SRE",
+      items: [
+        "self-hosted/operations/tracing",
+        "self-hosted/operations/api-access-control",
+        "self-hosted/operations/scalability",
+        "self-hosted/operations/observability",
+        "self-hosted/operations/logging",
```

Comment:
You have all the files in a single directory and ordered through filenames - everything's in place for you to turn this into an "autogenerated" category. :)